src/logcollector/read_snortfull.c

```diff
@@ -75,6 +75,7 @@ void *read_snortfull(logreader *lf, int *rc, int drop_it) {
                 else if ((str[2] == '/') && (str[5] == '-') && (q = strchr(str, ' '))) {
                     strncat(f_msg, "[Classification: Preprocessor] "
                             "[Priority: 3] ", f_msg_size);
+                    f_msg_size -= strlen("[Classification: Preprocessor] [Priority: 3] ");
```

Comment:
Use compile-time expressions for constants.
```suggestion
                    f_msg_size -= sizeof("[Classification: Preprocessor] [Priority: 3] ") - 1;
```